README.md

```diff
@@ -117,51 +177,41 @@ $ curl -s "search.mapzen.com/v1/reverse?size=1&point.lat=40.74358294846026&point
     ]
 }
 ```
+</details>
 
 ### How can I install my own instance of Pelias?
 
-For a quick preview of what pelias is like, check out our [Vagrant development environment](https://github.com/pelias/vagrant).
-Note that the Vagrant environment hasn't gotten a lot of updates lately, and it may not be working perfectly.
-
-To do a _real_ installation of Pelias for production use or serious development, check out our [full installation docs](http://pelias.io/install.html).
+To try out Pelias quickly, use our [Dockerfiles](https://github.com/pelias/dockerfiles/) setup. It uses Docker and docker-compose to allow you to quickly set up a Pelias instance for a small area (by default Portland, Oregon) in under 30 minutes.
 
-### How does it work?
-
-Magic! Well, like any geocoder, Pelias essentially combines [full text search](https://en.wikipedia.org/wiki/Full_text_search)
-techniques with knowledge of geography to quickly search over many millions of records, each of which representing some sort of location on Earth.
-
-The underlying architecture has three components:
-
-  * **import pipelines**: the pipelines used to filter, normalize, and ingest geographic datasets into the Pelias database.
-  * **database**: the underlying datastore that does all of the query heavy-lifting and powers our search results. We use
-    [ElasticSearch](https://www.elastic.co/).
-  * **API**: the thing that users interact with. A thin layer sitting on top of the datastore that implements additional
-    logic and features.
-
-Here's how they interact:
-
-![A diagram of the Pelias architecture.](https://cloud.githubusercontent.com/assets/4467604/6944539/3b1cdd0e-d862-11e4-995d-0b376caacad6.png)
+To do a _real_ installation of Pelias for production use or serious development, read our [full installation docs](http://pelias.io/install.html).
 
 ### What's it built with?
-Pelias itself (the import pipelines and API) is written in [Node](https://nodejs.org/), which makes it highly
+Pelias itself (the import pipelines and API) is written in [Node.js](https://nodejs.org/), which makes it highly
```

Comment:
Included supported Node versions. 6.0 and 8.0?

Reply:
I was thinking of adding a page specifically for dependencies. That way we can avoid specifying versions in every repository readme (they all get out of date anyway)